doc/nrf/ug_nrf52.rst

```diff
@@ -270,7 +269,39 @@ To perform a FOTA upgrade, complete the following steps:
 
 .. fota_upgrades_end
 
+.. fota_upgrades_matter_start
+
+FOTA over Matter
+================
+
+To perform a FOTA upgrade over Matter, use one of the following methods:
+
+* DFU over Bluetooth LE using either smartphone or PC command line tool.
```

Comment:
I'm not sure about this position. In Matter samples we allow user enable using DFU over Bluetooth, however it's not DFU over Matter in that case, but DFU over Bluetooth that is described in the section above. I'm afraid that users can misunderstand that in Matter it is possible to do the Matter DFU over Bluetooth that is not true. I think we could just skip writing about DFU over Bluetooth here.

Reply:
I see. I doubt removing the entry will solve the issue, as the Matter documentation page referenced 3 lines below still mentions DFU over Bluetooth LE, so we would be creating confusion in the opposite direction, so to say.

Reply:
@kkasperczyk-no , tried to address your concern in the latest push. Please check. The working of the header might solve the problem.

Reply:
Could we change the order of those points and somehow highlight that DFU over Matter is primary way to upgrade the device? The thing is that DFU over Matter is mandatory and always available. The DFU over Bluetooth is something optional that we added, as it can be valuable for somebody e.g. to upgrade the device firmware before it was commissioned to Matter network, so Bluetooth is the only available transport. Nevertheless the DFU over Bluetooth may not be present every application (even by default it will not be present).